.swiftlint.yml

```diff
@@ -1,2 +1,5 @@
 disabled_rules: # rule identifiers to exclude from running
 - large_tuple
+- function_body_length
```

Comment:
So why `function_body_length` here instead of in each file? This would apply even if someone decided to hand write some extension code, so putting it in the individual files is probably best.

Reply:
This is to disable the lint warning for the AutoRest swift project. Not the generated code.

Reply:
And we do have // swiftlint:disable function_body_length in the ServiceClientFile stencil.

Reply:
See I was mistaken :D